cloudinit/distros/bsd_util.py

```diff
@@ -0,0 +1,36 @@
+# This file is part of cloud-init. See LICENSE file for license information.
+
+import shlex
+
+from cloudinit import util
+
+
+def get_rc_config_value(key, fn='/etc/rc.conf'):
+    contents = {}
+    for line in util.load_file(fn).splitlines():
```

Comment:
It decided to just ignore the `if` blocks for now. I really doubt this will become a problem. If I'm wrong, we can still improve this parser. Or create a frankenstein-ish parser based on the RC one.

Reply:
Also, I like your approach to retrieve the value of a given key :+1:. I will reuse the logic.